app/src/main/java/com/example/marvelapp/data/service/response/CharacterResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.marvelapp.data.service.response
+
+import com.google.gson.annotations.SerializedName
+
+class CharacterResponse (
```

Comment:
These are all the character properties?

Reply:
No, in the future i will add the other properties, such as "comics" and "thumbnails"

Reply:
okay, but remember that you should not leave functionalities by the half